src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+
+var rightMap = ui.Map();
+var layer2 = ui.Map.Layer(ee.FeatureCollection([]), {'color': 'red'}, 'Points');
+
+// Link two layers together
+drawingTools1.onDraw(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.onEdit(ui.util.debounce(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+}, 100));
+drawingTools1.onErase(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+
+//Set map center using this long lat and zoom level
+rightMap.setCenter(33.78, -12.27, 11);
+
+// Make the default basemap the satellite layer
+rightMap.setOptions('satellite');
+
+//Specify the point as the only drawing tool 
+drawingTools1.setDrawModes(['point']);
+
+//Assign specific palettes to the cropland map
+var palettes = require('users/gena/packages:palettes');
+var palette = palettes.cmocean.Speed[7];
+
+// Use this empty image for paint().
+var empty = ee.Image().byte();
+
+var Admin_color = empty.paint({
+featureCollection: Admin,
+color: 1,
+width: 2
+});
+
+var EPA_color = empty.paint({
+featureCollection: Malawi_EPA,
+color: 1,
+width: 2
+});
+
+var TA_color = empty.paint({
+featureCollection: Malawi_TA,
+color: 1,
+width: 2
+});
+
+//Display the cropland map on the left panel
+var vis_crop = {min: 0, max: 1.0, palette: palette.slice(0,-2)};
+leftMap.addLayer(malawi_binary, vis_crop, 'Malawi cropland');
+leftMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+leftMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_EPA', false);
+leftMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_TA', false);
+
+//Visualize the Sentinel 2 high resolution image on the right panel
+var vis = {min: 0.0, max: 0.3, bands: ['B4', 'B3', 'B2']};
+rightMap.addLayer(Sentinel2, vis, 'Sentinel-2');
+rightMap.add(layer2);
+rightMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+rightMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_EPA', false);
+rightMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_TA', false);
+
+// Link the two panels
+var linker = ui.Map.Linker([leftMap, rightMap]);
+
+//Ceate a label for the data export
+var downloadUrl = ui.Label('', {
+  stretch: 'horizontal',
+  textAlign: 'left',
+  fontWeight: 'bold',
+  textDecoration: 'underline',
+  fontSize: '10',
+  height: '15px',
+});
+
+//Create a button to download all the data points as a feature collection
+var button = ui.Button('Get link for downloading points',function() {
+  var firstLayer = drawingTools1.layers().get(0);
+  var fc = firstLayer.getEeObject();
+  var sampled = upd_name.sample({
+    region: fc,
+    geometries: true,
+  });
+  //print(sampled);
+  sampled = sampled.map(function(feature) {
+    var point = feature.geometry().coordinates();
+    var long = point.get(0);
+    var lat = point.get(1);
+    return feature.set({
+      longitude: long,
+      latitude: lat,
+    });
```

Comment:
I don't really see the point of declaring separate variables here, why not just:
```javascript
var point = feature.geometry().coordinates();
return feature.set({
  longitude: point.get(0),
  latitude: point.get(1),
});
```